javascript/index.html

```diff
@@ -271,6 +271,15 @@ <h4>
                 </a>
             </li>
 
+             <li>
+                <a href="./answers/Smriti-Gupta-q9.html">
+                    <h4>
+                        Explain the difference between escape() and unescape() functions?
+                    </h4>
+                
```

Comment:
You haven't closed the 'a' tag.

Reply:
@smriti237 please resolve this ASAP

Reply:
Added the 'a' tag. I don't know how it vanished. I clearly checked it before pushing. Please review again..